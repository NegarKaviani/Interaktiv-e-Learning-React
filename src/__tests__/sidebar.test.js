import { render, screen } from '@testing-library/react'
import Sidebar from '../components/sidebar';

test("Example 1 renders successfully", () => {
    render(<Sidebar/>);

    const element = screen.getByText(/Money Laundering and Fraud Prevention/i);

    expect(element).toBeInTheDocument();
})