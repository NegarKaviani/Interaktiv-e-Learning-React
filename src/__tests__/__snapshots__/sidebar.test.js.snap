// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="sidebar w-full lg:w-3/12"
>
  <div
    className="box"
  >
    <div
      className="title-line bg-zinc-300 h-7 flex items-center text-xs pl-3 mb-5 font-semibold"
    >
      Navigation
    </div>
    <a
      className="course-nav block bg-white p-3 flex items-center mb-3"
      href="#/"
    >
      <div
        className="logo w-10 h-10 mr-3"
      >
        <img
          alt=""
          className="w-full h-full object-contain"
          src="fraud_prevention.png"
        />
      </div>
      <p
        className="text-sm font-semibold"
      >
        Money Laundering and Fraud Prevention
      </p>
    </a>
    <a
      className="course-nav block bg-white p-3 flex items-center mb-3"
      href="#/"
    >
      <div
        className="logo w-10 h-10 mr-3"
      >
        <img
          alt=""
          className="w-full h-full object-contain"
          src="it_security.png"
        />
      </div>
      <p
        className="text-sm font-semibold"
      >
        It security and information security
      </p>
    </a>
    <a
      className="course-nav block bg-white p-3 flex items-center mb-3"
      href="#/"
    >
      <div
        className="logo w-10 h-10 mr-3"
      >
        <img
          alt=""
          className="w-full h-full object-contain"
          src="data_protection.png"
        />
      </div>
      <p
        className="text-sm font-semibold"
      >
        Data Protection
      </p>
    </a>
    <a
      className="course-nav block bg-white p-3 flex items-center mb-3"
      href="#/"
    >
      <div
        className="logo w-10 h-10 mr-3"
      >
        <img
          alt="WpGH"
          className="w-full h-full object-contain"
          src="wphg.png"
        />
      </div>
      <p
        className="text-sm font-semibold"
      >
        WpHG-compliance
      </p>
    </a>
  </div>
  <div
    className="box"
  >
    <div
      className="title-line bg-zinc-300 h-7 flex items-center text-xs pl-3 mb-5 font-semibold"
    >
      Contact
    </div>
    <div
      className="contact-box pl-6 mb-4"
    >
      <p
        className="title font-bold"
      >
        Technical Support
      </p>
      <p
        className="person"
      >
        John Doe
      </p>
      <p
        className="info text-sky-800 font-semibold"
      >
        it@example.com
      </p>
    </div>
    <div
      className="contact-box pl-6 mb-4"
    >
      <p
        className="title font-bold"
      >
        Technical Support
      </p>
      <p
        className="person"
      >
        Jane Roe
      </p>
      <p
        className="info text-sky-800 font-semibold"
      >
        content@example.com
      </p>
    </div>
  </div>
</div>
`;
